nemo/collections/asr/data/feature_to_label.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, List, Optional
+
+import torch
+
+from nemo.collections.asr.parts import collections
+from nemo.core.classes import Dataset
+from nemo.core.neural_types import AcousticEncodedRepresentation, LabelsType, LengthsType, NeuralType
+from nemo.utils import logging
+
+
+def _feature_collate_fn(batch, pad_id):
+    """collate batch of feat sig, feat len, tokens, tokens len
```

Comment:
Add docstring for pad_id

Reply:
The pad_id here is not used/useful. Remove it for now, will add back and implement padding part in the future if necesarry.